### PR TITLE
Use yq4 in chart testing scripts

### PR DESCRIPTION
**Pull request by wurbanski**
**What this PR does / why we need it**:
Updates test-chart-rendering.sh to use yq4

**Which issue(s) this PR fixes**:

**What type of PR is this?**
Add one of the following kinds:
/kind bug
/kind chore

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: hack/test-chart-rendering.sh [hunk 1]
```diff
@@ -28,15 +28,15 @@ REALDIR="$(cd "$(dirname $(readlink -f "${BASH_SOURCE[0]}"))" && pwd)"
 source ${REALDIR}/lib.sh
 
 cd ${BASEDIR}
-chartname=$(yq '.name' Chart.yaml)
+chartname=$(yq e '.name' Chart.yaml)
```
Comment (xrstf) [inline hack/test-chart-rendering.sh:31]: I don't understand why this is necessary to explicitly call the eval function. All other scripts everywhere work fine without the `e`.

Comment (wurbanski) [reply hack/test-chart-rendering.sh:31]: me neither, but it didn't work locally for me and I wasn't sure if anyone actually ran this in a while ;-)

```
> test/test.sh 
Error: unknown command ".name" for "yq"
Run 'yq --help' for usage.
```

```
> yq -V
yq version 4.9.8
```

Comment (xrstf) [reply hack/test-chart-rendering.sh:31]: Your version is from Jul 8, 2021, maybe update yq 😁

Comment (wurbanski) [reply hack/test-chart-rendering.sh:31]: 🙈 yeah we can close it

**Pull request closed without merging**